wolfssl/wolfcrypt/asn.h

```diff
@@ -492,6 +492,24 @@ enum SignatureState {
     SIG_STATE_CHECK,
 };
 
+
+#ifdef HAVE_PK_CALLBACKS
+#ifdef HAVE_ECC
+    typedef int (*wc_CallbackEccVerify)(
+           const unsigned char* sig, unsigned int sigSz,
```

Comment:
The SSL layer callbacks use word32 vs unsigned int.  Makes portability a little harder.